### DISCUSSION TITLE
Dúvidas para iniciar na profissão

### DISCUSSION THREAD

**Discussion by costagondim** [frontendbr/forum#753] [Discussão]
Oi! Por acaso eu caí nesse fórum essa semana e achei muito legal o trampo de vocês. Não tenho conhecimento respeitável de programação ou T.I em geral mas tive muita curiosidade de saber o que fazem. Gostaria de saber se é difícil começar a trabalhar nessa área mas tem muita coisa que eu não entendo ainda, como o que é back/front-end, sou mais um curioso. 

Que atividades (cursos) eu posso fazer para aprender e poder dialogar com o grupo?
Qual faculdade é a mais comum para quem quer trabalhar desenvolvendo programas? 
Vale a pena entrar na faculdade? Sou meio velho pra ficar fazendo cursinho e as particulares aqui são bem ruins.
Esse sou eu: Tenho 20 anos e depois de me dar mal na faculdade de economia em BH por causa da crise e de problemas de saúde voltei para a casa dos meu pais em Rio das Ostras-RJ queria alguma coisa para fazer.

**Comment by costagondim** [comment]
Desculpa qqr erro de digitação, estou no ônibus e digitei meio no desespero

**Comment by kvnol** [comment]
Que massa, @costagondim. Tem muito conteúdo sobre como começar na área, o que eu indicaria a princípio são esses:

* [O que é Front-end e Back-end?](http://tableless.github.io/iniciantes/manual/obasico/o-que-front-back.html) - e todo o [Getting Started](http://tableless.github.io/iniciantes/) do @tableless
* [Do Zero a Herói Front-End](https://www.felipefialho.com/blog/2016/do-zero-a-heroi-do-front-end-parte-1) - tradução do @LFeh 

Bons estudos!

**Comment by costagondim** [comment]
E o que vcs são? Web-designers, Cientistas da Computação? Existe algum fórum mais indicado pra essas dúvidas mais bestas?
http://www.htmlandcssbook.com Eu tenho esse livro e pensei em estudar nele até o fim do ano, enquanto eu não tenho nenhuma obrigação. Ano que vem eu estou realmente em dúvida se devo passar um ano estudando para entrar em uma faculdade que eu nem sei se vai valer a pena ou entro de cabeça nesse lance, apesar de achar meio tarde para começar ( quem cresce ouvindo dos hackers de 13 anos sempre se acha velho pra entrar nessas áreas)

**Comment by kvnol** [comment]
@costagondim Aqui no fórum tem desde designers a engenheiros de computação. Se você com 20 anos (tenho 20, tb) se sente velho para entrar na área, se liga na história do mestre @maujor: http://devnaestrada.com.br/2016/08/18/entrevista-maujor.html

**Comment by marcusvcribeiro** [comment]
Na questão da faculdade, acho um desperdício de tempo, faça um tecnólogo de
2 anos e meio e fim.
O que vai contar é seu conhecimento mesmo, a não ser que você queira sair
do brasil, ai sim seria melhor ter um bacharel da vida por ai, porem se for
querer ficar sempre no brasil, estude, vire um autodidata e foque em
aprender o máximo possível da maneira certa.

E te digo, é uma área boa, se você for bom, jamais faltara emprego pra ti,
sendo que tem a oportunidade de desenvolver algo que de certo se você tiver
uma pegada empreendedora.
Onde eu vivo é mais difícil você entrar para frontend, pois brasília quem
da dinheiro para as empresas sao os backends da vida, devido os contratos
favorecerem isso.
Ai tu tem que ver qual tua afinidade com as áreas de TI e estudar igual um
louco.

Em 5 de outubro de 2017 16:17, costagondim <notifications@github.com>
escreveu:

**Comment by alinebastos** [comment]
Tu jamais vai ser velho pra iniciar na nossa profissão ou na faculdade. Eu to começando Computer Science agora com 42. Comecei na carreira com 35.
Faculdade depende muito de ti mesmo, de como tu te sente melhor aprendendo.
Tem gente que prefere aprender sozinho, aí pega um tecnólogo, que é mais rápido, pelo diploma.
Tem gente que nem faz faculdade, pq na maioria das empresas nem pedem diploma, e na web tu consegue aprender tudo!
Eu já to querendo aprender toda a base mesmo, a lógica, a matemática, por trás ds linguagens, por isso resolvi começar CS agora numa faculdade 100% online, a UoPeople.
Além dos artigos que já te recomendaram ali em cima, te recomendo também a entrar no grupo do Training Center no Slack, ali tem bastante gente fera e bastante gente iniciando na carreira, pegando dicas, trocando ideias, de todas as áreas da TI.
https://ctgroups.herokuapp.com/
;)

**Comment by costagondim** [comment]
Moro há pouco tempo na cidade de Macaé. Vocês recomendam algum curso por aqui ou no Rio/BH?

**Comment by costagondim** [comment]
Estou pensando, caso eu goste, de fazer esse curso online da FIAP

**Comment by delisola** [comment]
Não discordo do que o Marcus Vinicus ( nome legal ) falou. Faculdade não é
pré-requisito para começar ou trabalhar com a profissão de TI, no entanto,
você pode se deparar com alguma oportunidade que exija um curso superior e
ai você estaria perdendo essa oportunidade. Você tem que ser uma excelente
profissional ( com bastante conhecimento e profissionalismo ) para virar as
costas para faculdade, e na pior das hipóteses a faculdade vai te ensinar
muita coisa sim, apesar de que pra trabalhar com TI a gente aprende mais
apanhando e estudando por conta própria.

TI é um mercado muito bacana de trabalhar, se você for uma pessoa que se
interessa em aprender uma coisa nova TODOS os dias ( Javascript dominando
pesado ), essa profissão é pra ti.

On Thu, Oct 5, 2017 at 5:26 PM, costagondim <notifications@github.com>
wrote:

> Moro há pouco tempo na cidade de Macaé. Vocês recomendam algum curso por
> aqui ou no Rio/BH?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/753#issuecomment-334582148>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH0k8Mo9wCYrCDugQQx7X6xQ83d8m9-4ks5spTuPgaJpZM4Pviys>
> .
>



-- 
Antonio Marcos
61 9178.3088

**Comment by thismarcoantonio** [comment]
Que legal @costagondim, no meu caso foi o contrário, vim do RJ pra BH. Enfim, primeiramente essa é uma área que você vai receber palpite e opiniões de todos os lados (sim, estou fazendo isso agora), basta a você discernir o que abstrair ou não, essa área é sempre avaliada pessoalmente por cada um de nós. No meu caso, comecei a fazer Sistemas de Informação, tive que desistir da faculdade por motivos particulares e agora estou tentando entrar no mercado. Acredito que faculdade não seja um ponto necessário, é mais coisa de status, já que muita coisa que comecei a ver na faculdade e a grade apresentavam ser desatualizadas.

**Comment by sbvorpagel** [comment]
@costagondim se você quer algo que te guie do básico, talvez um curso como o do alura seja um bom início, pois lá você teria uma visão geral da computação (Front, back, mobile, IA etc). 

Obs:. não sei se pode falar de empresas aqui, e se não puder, eu excluo esse comentário.
